src/main/kotlin/com/exactpro/th2/check1/rule/AbstractCheckTask.kt

```diff
@@ -459,6 +472,22 @@ abstract class AbstractCheckTask(
         }
     }
 
+    protected fun ProtoToIMessageConverter.fromProtoPreFilter(protoPreMessageFilter: RootMessageFilter,
+                                                              messageName: String = protoPreMessageFilter.messageType): IMessage {
+        try {
+            return fromProtoFilter(protoPreMessageFilter.messageFilter, messageName)
+        } catch (e: PatternSyntaxException) {
```

Comment:
There might be other exceptions (e.g. NuberFormatException). So, I think we should catch any exception that is thrown during the filter creation